Alignment/LaserAlignment/python/HLTLaserAlignmentEventFilter_cfi.py

```diff
@@ -0,0 +1,18 @@
+import FWCore.ParameterSet.Config as cms
+
+
+from Alignment.LaserAlignment.LaserAlignmentDefs_cff import *
```

Comment:
ok, but then the users should call the cff. cfis there there to define a module (and this module doesn't appear to need the constants)